modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -175,6 +177,31 @@ Method Label Options Color:: Specifies the OPTIONS method label color.
 Method Label Head Background Color:: Specifies the HEAD method label background color.
 Method Label Head Color:: Specifies the HEAD method label color.
 
+If an *API Console* navigation item such as an API endpoint or method has a long name, it wraps around to a second line. By default, the line breaks at the end of a word. To configure the line break behavior:
```

Comment:
```suggestion
If an *API Console* navigation item such as an API endpoint or method has a long name, that name wraps around to a second line. By default, the line breaks at the end of a word. To configure the line break behavior:
```
WHY does the audience have to configure this if the default breaks at the end of a word: is this not the desirable outcome?

Reply:
In most cases customers do want the default, but a few customers sometimes want different behavior, so the new version adds this customization. Reasons for wanting different behavior depend on how you're organizing the page, and whether you're working with Chinese/Japanese/Korean text. More info at the CSS reference for the `word-break` property:
https://www.w3schools.com/cssref/css3_pr_word-break.asp

I updated the text to use "the name" and not "it", and clarified the optional customization.